cypress.config.js

```diff
@@ -1,18 +1,23 @@
 const { defineConfig } = require('cypress');
 
-const { isProd } = require('./utils/config-utils');
+const ENV_DEV = 'development';
+const ENV_PROD = 'production';
+const isDev = process.env.NODE_ENV === ENV_DEV;
+const isProd = process.env.NODE_ENV === ENV_PROD;
```

Comment:
Moved all instances of the prod check here since only Cypress cares about it now.